core/src/main/java/org/apache/spark/unsafe/map/BytesToBytesMap.java

```diff
@@ -678,20 +747,31 @@ private void allocate(int capacity) {
   }
 
   /**
+   * Free the memory used by longArray.
+   */
+  public void freeArray() {
```

Comment:
This also nulls out the bitset, but i guess we don't do memory accounting for it since it's so relatively small?